documentation/gitbook/how-to-run/quick-start-guide.md

```diff
@@ -170,29 +86,29 @@ To get started, let's start with just running a forward simulation (non-inferenc
 
 ### Non-inference run
 
-Stay in the `PROJECT_PATH` folder, and run a simulation directly from forward-simulation Python package gempyor. Call `flepimop simulate` providing the name of the configuration file you want to run. For example here, we use `config_sample_2pop.yml` from _flepimop\_sample_.
+Stay in the `PROJECT_PATH` folder, and run a simulation directly from forward-simulation Python package `gempyor`. Call `flepimop simulate` providing the name of the configuration file you want to run. For example here, we use `config_sample_2pop.yml`.
 
 ```
 flepimop simulate config_sample_2pop.yml
 ```
 
-This will produce a `model_output` folder, which you can look using provided post-processing functions and scripts.
+This will produce a `model_output` folder, which you can look at using provided post-processing functions and scripts.
 
-We recommend using `model_output_notebook.Rmd` from _flepimop\_sample_ as a starting point to interact with your model outputs. First, modify the yaml preamble in the notebook, then knit this markdown. This will produce some nice plots of the prevalence of infection states over time. You can edit this markdown to produce any figures you'd like to explore your model output.
+We recommend using `model_output_notebook.Rmd` as a starting point to interact with your model outputs. First, modify the YAML preamble in the notebook (make sure the configuration file listed matches the one used in simulation), then knit this markdown. This will produce plots of the prevalence of infection states over time. You can edit this markdown to produce any figures you'd like to explore your model output.
 
-The first time you run all this, it's , it's better to run each command individually as described above to be sure each exits successfully. However, eventually you can **put all these steps together in a script**, like below
+For your first `flepiMoP` run, it's better to run each command individually as described above to be sure each exits successfully. However, eventually you can **put all these steps together in a script**, seen below:
 
 ```
 export FLEPI_PATH=/Users/YourName/Github/flepiMoP
-export PROJECT_PATH=/Users/YourName/Github/flepiMoP_sample
+export PROJECT_PATH=/Users/YourName/Github/flepiMoP/examples/tutorials
 cd $FLEPI_PATH
 pip install --no-deps -e flepimop/gempyor_pkg/
```

Comment:
I think we can get rid of these two lines since we take care of setting up the environment in the prior page with `./bin/local_install_or_update` script right?